shells/default/default.nix

```diff
@@ -15,6 +15,7 @@
   podman,
   mitmproxy,
   cargo-nextest,
+  daemonize,
```

Comment:
Why do we provide the package here and in `flox-cli-tests` when it's used from a Flox environment?

Reply:
Just adding it to `flox-cli-tests` doesn't make it available in your devshell if you want to play around with it or test something interactively. Not strictly necessary, but helpful when writing tests that use it.

Reply:
Doesn't strictly need to be added to `flox-cli-tests` anymore though.